packages/trip-details/src/styled.ts

```diff
@@ -1,5 +1,14 @@
 import styled from "styled-components";
 
+export const InvisibleA11yLabel = styled.span`
+  display: inline-block;
+  clip: rect(0, 0, 0, 0);
+  height: 0;
+  overflow: hidden;
+  position: absolute;
+  width: 0;
+`;
+
```

Comment:
We do already have this in itinerary body. Should there be one that gets exported from a more central location?

Reply:
Yeah you're right. I just really didn't want to go through the pain of updating several packages.